app/models/content_item.rb

```diff
@@ -46,7 +46,8 @@ def fetch_content_item
   # ***  OVER CONTENT STORE REQUESTS  ***
   # *************************************
   FINDERS_IN_DEVELOPMENT = {
-    "/review-search/all" => 'all_content'
+    "/review-search/all" => 'all_content',
+    "/review-search/news-and-communications" => 'news_and_communications',
```

Comment:
Is this right? or a leftover debug thing?

Reply:
It'll allow you to view the changes to the pages in the Heroku app and in development, but won't be available to the public.